h/services/rename_user.py

```diff
@@ -88,18 +88,9 @@ def _fetch_annotations(self, userid):
         )
 
 
-def make_indexer(request):
-    def _reindex(ids):
-        if not ids:
-            return
-
-        request.tm.commit()
-        indexer = index.BatchIndexer(request.db, request.es, request)
-        indexer.index(ids)
```

Comment:
Again, we had indexing behavior distributed. It's much nicer if this service doesn't have to worry about these details.